packages/flutter/lib/src/gestures/recognizer.dart

```diff
@@ -495,6 +495,7 @@ abstract class PrimaryPointerGestureRecognizer extends OneSequenceGestureRecogni
 
   @override
   void acceptGesture(int pointer) {
+    _stopTimer();
```

Comment:
Can you also update the documentation on deadline to indicate when didExceedDeadline will not fire?

Reply:
Done!